monkey-runner/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Oct 20 10:41:37 BST 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-all.zip
```

Comment:
Updated to the latest version and it seems there are no issues. I added the wrapper task so that I could specify the version in the `dependencies.gradle` once rather than in each place separately:

https://github.com/novoda/spikes/pull/144/commits/f09336b4fffdbc929d6fe311b1cc07a13afe4566